web/stories/pages/posts/slow-mock-env.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import {
+    openedSlowPost,
+    slowPosts,
+    totalSlowPages,
+    totalSlowPosts
+  } from '@slowreader/core'
+  import type { PostValue, SlowPostsValue } from '@slowreader/core'
+  import { onMount } from 'svelte'
+
+  import { isWritableAtom } from '../../../../core/lib/stores.js'
+  import Slow from '../../../pages/slow.svelte'
+
+  export let state: SlowPostsValue = {
+    isLoading: true
+  }
+  export let showPagination = false
+  export let post: PostValue | undefined = undefined
+
+  onMount(() => {
+    if (
+      isWritableAtom(slowPosts) &&
+      isWritableAtom(totalSlowPages) &&
+      isWritableAtom(totalSlowPosts) &&
+      isWritableAtom(openedSlowPost)
+    ) {
+      slowPosts.set(state)
```

Comment:
Can we move it to `scene.svelte`?

We will need to test `slowPosts` stores in other stories too later. Also, right now all stores mocking are in the same place, which is very flexible.

Reply:
Sure